### DISCUSSION TITLE
Understanding the workflow

### DISCUSSION THREAD

**Discussion by scienception** [PGScatalog/pgsc_calc#130] [Q&A] [answered]
I wanted to know if you had any documentation on the workflow of this software package. And more specifically, in which script are the PGS computed from the weights and dosage? 

Thanks!

**Comment by smlmbrt** [comment, marked as answer]
Some of that information will be in the `results/pipeline_info` directory, specifically the image of the DAG for your data. The actual scoring is done with the [`PLINK2_SCORE`](https://github.com/PGScatalog/pgsc_calc/blob/main/modules/local/plink2_score.nf) module which uses plink.

**Comment by scienception** [comment]
You don't use "thresholding" in your method? (I didn't see the `--q-score-range` flag). I guess my question is whether this is the C+T method or another one.

**Reply by smlmbrt** [reply]
No, the pipeline is meant to calculate final PGS (lists of variants and weights). We consider p-value thresholding to be more in the arena of PGS development (for which other tools are optimised).
